timelock-server/src/integTest/resources/paxosSingleServer.ftl

```diff
@@ -30,7 +30,7 @@ server:
   minThreads: 1
   maxThreads: 200
   applicationConnectors:
-    - type: h2
+    - type: https
```

Comment:
apparently this makes it work locally, idk if we want to necessarily push to develop. @sudiksha27